controllers/digital-funding/views/full-eligibility.njk

```diff
@@ -0,0 +1,199 @@
+                    <li>Products of services that cannot show considerable evidence of user research in their composition</li>
+                    <li>Products or services that duplicate existing services with no substantial innovation</li>
+                </ul>
+
+                <p><a href="/funding/programmes/digital-funding/strand-1/eligibility"
+                    class="btn btn--medium accent--{{ pageAccent }} a--btn u-margin-bottom">Request a call about strand one</a></p>
+
+
+
+                <h2>Success factors for strand two</h2>
+
+                <p><strong>We are already highly digital, and want to scale our impact</strong></p>
+                <p>Grants of up to £500,000 for tech for good organisations that have already launched promising services</p>
+
+                <p>Successful proposals will:</p>
+
+                <ul>
+                    <li>Demonstrate a firm commitment by organisational leadership to changing the way that services are delivered.</li>
+                    <li>Contain a plan that has been developed in accordance with good digital practices.</li>
+                    <li>Supply evidence that the organisation has already invested in digital, prior to this proposal.</li>
+                    <li>Show a commitment to working generously, sharing what you learn and the products or services you produce with others.</li>
+                </ul>
+
+                <p>We are not looking for:</p>
+                <ul>
+                    <li>Proposals that contain a highly detailed plan setting out every detail of your forthcoming transformation. Instead we welcome proposals that clearly differentiate between what you know now, and what you will learn along the journey.</li>
+                    <li>Proposals that are driven by a desire to use technology, rather than by a desire to make change.</li>
+                    <li>Proposals that are based around core budget shortfalls.</li>
+                </ul>
+
+                <p><a href="/funding/programmes/digital-funding/strand-2/eligibility"
+                    class="btn btn--medium accent--{{ pageAccent }} a--btn">Request a call about strand two</a></p>
+            {% endcall %}
+
+            {% call contentBox(pageAccent, id = "segment-2") %}
+                <h2>Eligibility criteria</h2>
+
+                <p><strong>The following eligibility criteria apply to both strands of funding. There are specific criteria for each strand below this section.</strong></p>
+
+                <p>You can apply if you’re a:</p>
+
+                <ul>
+                    <li>voluntary and community organisation</li>
+                    <li>registered charity</li>
+                    <li>social enterprise</li>
+                    <li>group of organisations, as long as they are led by a voluntary and community organisations or social enterprise</li>
+                    <li>Community interest companies (with two or more directors).</li>
```

Comment:
Should this be lowercase C for "Community", as well as singular (eg. "company")?